handlers/schemas/UploadVOD.yaml

```diff
@@ -19,11 +19,11 @@ properties:
     description:
       Force to use a specific strategy in the Catalyst pipeline. If not
       specified, the default strategy that Catalyst is configured for will be
-      used. This field only available for admin users, and is only used for E2E
-      testing.
+      used. This field only available for admin users.
     enum:
       - catalyst
       - external
+      - ffmpeglivepeer
```

Comment:
i think `livepeer` sounds better without having to reference ffmpeg. What if we build some parts of the pipeline with gstreamer in the future?

Reply:
I don't love catalyst_v2 just because of the mental layer of indirection between name and implementation (plus assumption that one definitely supersedes the other) but I think catalyst_ffmpeg is nice - let's go with that

Reply:
@emranemran that's a good point, but I think in that case we'll do the same approach of a parallel pipeline that eventually gets merged in / becomes dominant